xarray/tests/test_interp.py

```diff
@@ -132,29 +132,55 @@ def func(obj, new_x):
     assert_allclose(actual, expected)
 
 
-@pytest.mark.parametrize("use_dask", [False, True])
-def test_interpolate_vectorize(use_dask: bool) -> None:
+@pytest.mark.parametrize(
+    "use_dask, method",
+    (
+        (False, "linear"),
+        (False, "akima"),
+        (False, "makima"),
+        (True, "linear"),
+        (True, "akima"),
+    ),
+)
+def test_interpolate_vectorize(use_dask: bool, method: str) -> None:
     if not has_scipy:
         pytest.skip("scipy is not installed.")
```

Comment:
```suggestion
```

driveby cleanup. The `requires_scipy`decorator will skip if scipy isn't available. Import it  from `xarray.tests` if it's not present.